4-storage/relational-database-shredder/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,8 @@
+log4j.rootCategory=WARN, file
```

Comment:
It isn't required, without it everything is logged to stdout and clutters the test output however.